lpa-submissions-e2e-tests/cypress/support/common/clickFooterLink.js

```diff
@@ -0,0 +1,18 @@
+import {
+  accessibility,
+  cookies,
+  privacy,
+  termsAndConditions,
+} from '../PageObjects/footerPageObjects';
+
+export const clickFooterLink = (footerLink) => {
+  if (footerLink === 'Privacy') {
+    privacy().click();
```

Comment:
These can have the objects directly. Having an additional import seems to do little to add modularity here

Reply:
This is the industry standard for creating automation framework. If required, I can remove it but we will be deviating from the general practice and is against the SOLID principles.

Reply:
This is against cypress best practice though. I don't profess to be an automation framework specialist, however the framework itself suggests this is not best practice.

I don't see how we are breaking SOLID principles by not doing this. Having layers of interdependency increases cognitive load when working on them, and makes changes far harder to implement and track.

Reply:
updated as per review comments